go/conversation/forms.py

```diff
@@ -54,3 +54,35 @@ class ReplyToMessageForm(forms.Form):
         label='Reply Message',
         required=True,
         widget=forms.Textarea)
+
+
+class MessageDownloadForm(forms.Form):
+
+    CUSTOM_DATE_FORMAT = '%Y/%m/%d'
+    _LOCALS = locals()
+
+    format = forms.ChoiceField(
+        required=True, initial='csv',
+        choices=[
+            ('csv', 'csv'), ('json', 'json'),
+        ])
+
+    direction = forms.ChoiceField(
+        required=True, initial='inbound',
+        choices=[
+            ('inbound', 'inbound'), ('outbound', 'outbound'),
+        ])
+
+    _LOCALS['date-preset'] = forms.ChoiceField(
```

Comment:
If it is just the `name` attribute that would be affected, then I think using `date_to`, etc makes sense, thanks.